core/templates/pages/topic-editor-page/editor-tab/topic-editor-stories-list.directive.html

```diff
@@ -75,6 +77,10 @@
     background-color: #ededed;
   }
 
+  stories-list .delete-icon {
```

Comment:
Do we still need this change? (Considering the anchor tag will have the `cursor: pointer` style, right?) 

(Ditto elsewhere)

Reply:
Yes, you are right, we do not need this change anymore.
So should I use the anchor tag at the other two places too?

Reply:
Yes, and please add a screenshot of the UI changes (before and after).

Reply:
Done. The screenshots don't show the mouse pointer, so they not of any use here.
Thanks for the insight into this!